fvm/bootstrap.go

```diff
@@ -781,6 +781,8 @@ func (b *bootstrapExecutor) setStakingAllowlist(
 }
 
 func (b *bootstrapExecutor) setupEVM(service flow.Address) {
+
+	b.createAccount(nil) // account for storage
```

Comment:
Is the account later on referred to by address? Is this order dependent?

Reply:
this is the address we use for storage of data. (we have the index, so we don't need to capture the address) It is order-dependent but other initial addresses are also like this one, I might refactor this soon.